ui/src/views/Creator.vue

```diff
@@ -37,7 +31,11 @@
           />
         </div>
         <div class="button-bar" id="creator-button-bar">
-          <Button v-if="hasQueryDefinition" icon="pi pi-bolt" label="Test query" severity="help" @click="testQuery" />
+          <QuickQuery v-if="isObjectHasKeys(editorEntity, [IM.DEFINITION])" :query="editorEntity[IM.DEFINITION]">
```

Comment:
I would like it if this button/functionality was contained in the query/definition specific components, not a v-if in the creator/editor. It's a slippery slope adding v-if's for specific keys/shapes in the editor root.

Also from a ui perspective, running the query from the query/definition section keeps the button's use scoped to that section.